modules/libraries/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../tsconfig.shared.json",
   "compilerOptions": {
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "skipLibCheck": true
```

Comment:
Why remove the type checking?

Reply:
Libraries module also has NPM, which declares node.d.ts, but it conflicts with our set version of node and it complains about a new declaration overriding a previous one but with a different signature. I tried a couple of solutions to fix this, to no avail

Reply:
I have no issues building the module without the skipLibCheck on my end

![Screenshot from 2021-12-03 12-42-43](https://user-images.githubusercontent.com/9640576/144648335-f736bce4-1e5c-4c4d-a54d-5b083ba3b324.png)

Reply:
Ah you're right, it only happens with yarn2. I'm taking baby steps to bring the repository closer to an "ideal state" before switching from yarn 1 and yarn 2. I get this with yarn 2. 

![image](https://user-images.githubusercontent.com/42552874/144652724-570a9418-5124-4d62-86d4-6621e94d5a6c.png)

I think we can keep that flag for now, we'll scrap the libraries module eventually, theres a better alternative